ui/raidboss/raidboss_config.ts

```diff
@@ -209,7 +209,7 @@ const kDetailKeys = {
       de: 'alarm',
       fr: 'alarme',
       ja: '警報',
-      cn: '警报文本',
+      cn: '致命级文本',
```

Comment:
```suggestion
      cn: '致命级',
```

cause it is not only refer to "text", but also TTS.